engines/bops_config/app/controllers/bops_config/application_types_controller.rb

```diff
@@ -6,6 +6,16 @@ class ApplicationTypesController < ApplicationController
     before_action :set_application_types, only: %i[index]
     before_action :set_application_type, only: %i[show edit update]
 
+    audit :create, :update,
+      unless: -> { @application_type.changed? },
+      payload: -> {
+        {
+          application_type: @application_type.audit_attributes,
+          changes: @application_type.audit_changes,
+          automated: false
+        }
+      }
```

Comment:
This is the per-audit event payload that's merged with the shared payload - it's also executed in the context of the controller.